changelogs/fragments/6395.yml

```
@@ -0,0 +1,2 @@
+feat:
+- [MDS] Do not support import data source object to Local cluster when not enable data source ([#6395](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6395))
```

I am not sure about the context but is this maybe
```suggestion
- [MDS] Prevent importing of data source object when MDS is not enabled ([#6395](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6395))
```